script.js

```diff
@@ -52,4 +52,14 @@ function clean() {
 function back() {
     var exp = document.form.textview.value;
     document.form.textview.value = exp.substring(0, exp.length-1);
+}
+function sinFunc(){
+    var a = document.form.textview.value;
+    
+    document.form.textview.value = Math.sin(a);
+}
+function sqrFunc(){
+    var a = document.form.textview.value;
+    
+    document.form.textview.value = Math.sqrt(a);
 }
```

Comment:
Git likes files to end with a newline. To get rid of this "No newline at end of file" message just add a newline (press ENTER) at the bottom of the file so git will be happy :P